src/pages/waiting-room/components/cbt-number/cbt-number.ts

```diff
@@ -22,8 +22,11 @@ export class CBTNumberComponent implements OnChanges {
 
   ngOnChanges(): void {
     if (!this.formControl) {
-      this.formControl = new FormControl(null);
+      this.formControl = new FormControl(null, [
+        Validators.pattern('[0-9]{0,10}'),
```

Comment:
This means the cbt number can only be up to 10 characters long, is there any chance that it could be over this amount?

Reply:
removed validation pending future task